docs/source/dataframe.rst

```diff
@@ -8,198 +8,181 @@ Dask DataFrame
    :maxdepth: 1
    :hidden:
 
-   dataframe-create.rst
+   Load and Save Data <dataframe-create.rst>
+   Internal Design <dataframe-design.rst>
    Best Practices <dataframe-best-practices.rst>
-   dataframe-design.rst
-   dataframe-groupby.rst
-   dataframe-joins.rst
-   dataframe-indexing.rst
-   dataframe-categoricals.rst
-   dataframe-extend.rst
-   dataframe-parquet.rst
-   dataframe-hive.rst
-   dataframe-sql.rst
-   dataframe-api.rst
-
-A Dask DataFrame is a large parallel DataFrame composed of many smaller pandas
-DataFrames, split along the index.  These pandas DataFrames may live on disk
-for larger-than-memory computing on a single machine, or on many different
-machines in a cluster.  One Dask DataFrame operation triggers many operations
-on the constituent pandas DataFrames.
+   API <dataframe-api.rst>
+   dataframe-extra.rst
 
-.. raw:: html
+.. grid:: 1 1 2 2
 
-   <iframe width="560"
-           height="315"
-           src="https://www.youtube.com/embed/AT2XtFehFSQ"
-           style="margin: 0 auto 20px auto; display: block;"
-           frameborder="0"
-           allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-           allowfullscreen></iframe>
+   .. grid-item::
+      :columns: 12 12 8 8
 
-Examples
---------
+      Dask DataFrame helps you process large tabular data by parallelizing pandas,
+      either on your laptop for larger-than-memory computing, or on a distributed
+      cluster of computers.
 
-Visit https://examples.dask.org/dataframe.html to see and run examples using
-Dask DataFrame.
+      -  **Just pandas:** Dask DataFrames are a collection of many pandas DataFrames.
 
+         The API is the same.  The execution is the same.
```

Comment:
This is a much more streamlined, user-centric wording that before. I like that. 

However, "The API is the same" isn't totally correct (it's aspirational) and "The execution is the same" is a bit too generic. I think the intent here is to say something like "We're just doing a bunch of parallel pandas operations for you. It's a complicated task graph, but really it's just lots of pandas, which you're familiar with, happening." I think the changes here convey this, but in a way that may be _too_ streamlined. 

I think the change here is an improvement on what we currently have, and I don't have a better suggestion at the moment, so let's go with the current change here. That said, I wanted to flag this as something that I may circle back to if I think of something that's still concise, but fits well here.